care/facility/api/viewsets/asset.py

```diff
@@ -117,6 +157,22 @@ def set_default_user_location(self, request, *args, **kwargs):
         except:
             raise Http404
 
+    # Asset Integration API
+    @action(detail=False, methods=["POST"])
```

Comment:
detail should be true right? the asset can be identified, authorized, and fetched by Django that way.

Reply:
Fiex